web/js/script.js

```diff
@@ -0,0 +1,7 @@
+const form = document.getElementById('form');
+
+// const submit = document.getElementById('button');
```

Comment:
If we don't need this code, remove it. If we need it, commit it only when needed.